### DISCUSSION TITLE
Export Pure HTML,CSS & JS

### DISCUSSION THREAD

**Discussion by antonnifo** [webstudio-is/webstudio-community#72] [General]
Hello team, I'm looking for a way to get HTML, CSS, and JS code from a design. Is there a method to do this, particularly for use in Django?

**Comment by kof** [comment]
I am guessing you are asking how to exports HTML/CSS/JS and then use it from Django ... I think not at the moment. We are creating JSX output and Remix application.

Pure HTML/CSS output will be possible at some point with limitations as you can't keep data fetching etc like that. Also community will be able to create tools for convenient work with other frameworks than Remix, but its not going to be our responsibility to do that for every framework. If you are keen to try, I welcome you to look into our CLI and the build we are creating today.

**Reply by mmiroshnikov** [reply]
I will absolutely consider moving from WF to WS, but a clean HTML+CSS export is crucial to my workflow. I'll keep an eye on this feature and hopefully you or the community will do something about it 🙏

**Comment by youngjid** [comment]
Web studio has just announced their latest feature that enables you to export Static Sites (HTML, CSS and JS).
Which is great for some people who just want their simple design export without dynamic data.

You can check this feature out on the Webstudio website on all plans even Free Plan.
Good Luck.

**Comment by Sonicprint** [comment]
Hey Guys! Im facing a little issue. was wondering if anyone else has faced the same problem...

I have made a navbar, that has nav items, the navitems each have a link and a horizontal line, that is initially out of frame, (overflow=hidden), I've written a small JS snippet that checks for mouse enter and exit, and accordingly the line moves to view. 

the problem: This works when i test it on the editor but when i publish it, the site is broken, with the navitems invisible and the lines showning. When i export as static, the site is not broken but the hoverline effect doesnt work. (When client only=disabled), the export is also broken when (client only=enabled)

I'm not sure what's going on here...
